apps/dokploy/next.config.mjs

```diff
@@ -38,6 +39,13 @@ const nextConfig = {
 				],
 			}),
 		);
+		config.plugins.push(new SmartLocalePlugin({
+			inputDirectory: "./i18n/locales/",
+			outputDirectory: "./i18n/InternationalizationKit/interface/",
+			outputImportDirectory: "./i18n/",
+			defaultLocale: "en-us",
+			fileType: "json",
+		}));
 		return config;
```

Comment:
I pushed a feature tag to dockerhub to test this and I'm getting this error which I suspect is coming from this
![image](https://github.com/user-attachments/assets/598eef4f-bf35-4c00-a86a-1301dc263380)